api/tests/integration/ArticleBookmarkRateTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the Spira framework.
+ *
+ * @link https://github.com/spira/spira
+ *
+ * For the full copyright and license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+use App\Models\Article;
+
+class ArticleBookmarkRateTest extends TestCase
+{
+    public function testSimpleRate()
+    {
+        $user = $this->createUser();
+        $article = $this->getFactory(Article::class)->create();
+        $rateData = $this->getFactory(\Spira\Rate\Model\Rating::class)->transformed();
+
+        $token = $this->tokenFromUser($user);
+        $this->withAuthorization('Bearer '.$token)->putJson('articles/'.$article->article_id.'/rate/'.$rateData['ratingId'], $rateData);
+
+        $this->assertResponseStatus(201);
+
+        $this->assertEquals($user->ratedArticles->first()->article_id, $article->article_id);
+    }
```

Comment:
Assert that the rating value (1-5) that was generated by the factory was actually saved.